kive/librarian/management/commands/find_orphans.py

```diff
@@ -41,22 +39,20 @@ def find_orphans():
         return orphans
 
     @staticmethod
-    def display_orphans(orphans, root_path=settings.MEDIA_ROOT, verbosity=1):
+    def display_orphans(orphans, verbosity=1):
         norphans = len(orphans)
```

Comment:
It looks like norphans is only used in one branch of the if, so please calculate it there.
Also, use orphans.count() instead of len(orphans), because it does the counting in the database.